PrestaShop/PrestaShop
# Use "brand" as column heading

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This fixes the example import file products_example.csv to use the format expected during import process. It replaces "Manufacturer" by "Brand" in the CSV example file. Consequently in the import process, the data column and the file columns are matched. See the "How to test" to better understand.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See below
| UI Tests          | https://github.com/kpodemski/ga.tests.ui.pr/actions/runs/10109614030
| Fixed issue or discussion?     | Progress on #36117, related to [https://github.com/PrestaShop/PrestaShop/discussions/ {discussion number here}](https://github.com/PrestaShop/PrestaShop/discussions/34978) but does not completely fix them
| Related PRs       | None

# How to test?

## In order to understand the problem

1. Go to backoffice import page, download the "Sample product file"
2. This is a CSV file named `products_import.csv`. Keep it.
3. Now you are in backoffice import page. Fill the form:
- `* What do you want to import?` input "Products"
- `Select a file to import` input the products_import.csv file you downloaded
4. Click on "Next step"
5. On the next screen, use the arrows to browse the file and see the column "Manufacturer" in the file. It is used to match the "Brand" column in the data matching

![Capture d’écran 2024-06-07 à 15 54 51](https://github.com/PrestaShop/PrestaShop/assets/3830050/1bc623ab-0bfc-4672-a4fb-a89990dc26d7)

## In order to verify the fix

1. Use the code from this PR. Go to backoffice import page, download the "Sample product file"
2. This is a CSV file named `products_import.csv`. Keep it.
3. Now you are in backoffice import page. Fill the form:
- `* What do you want to import?` input "Products"
- `Select a file to import` input the products_import.csv file you downloaded
4. Click on "Next step"
5. On the next screen, use the arrows to browse the file and see the column "Brand" in the file. It is used to match the "Brand" column in the data matching. The two match!

![Capture d’écran 2024-06-07 à 15 55 14](https://github.com/PrestaShop/PrestaShop/assets/3830050/0cfaa7df-f3c2-49dd-b179-8d9691aa9c27)
